### PR TITLE
Feat/homepage

### PR DESCRIPTION
**Pull request by dhafinn**
**Summary:**
1. Finished slicing for homepage
2. Circular and linear progress bar for statistics
3. Currently use mockup data that mimics possible API response

### CODE REVIEW

File: app/src/main/AndroidManifest.xml
Comment (rifqifarelmuhammad) [inline app/src/main/AndroidManifest.xml:1]: Pin can you change the launcher activity to splash? Considering that checking the user session will be done in the splash activity

**Comment by rifqifarelmuhammad** [comment]
My phone is using dark mode and I feel like the display for its dark mode needs to be forced into light mode?
image
image

File: app/src/main/AndroidManifest.xml
Comment (dhafinn) [reply app/src/main/AndroidManifest.xml:1]: walach lupa balikin pak

**Comment by dhafinn** [comment]
**Changes:**
1. Return launcher intent to splash screen
2. Add strictly colors in card
3. Add counting-like animation in inventory card

cc: @rifqifarelmuhammad

**Comment by rifqifarelmuhammad** [comment]
LGTM, I really like the animation on the progress bar. Let's merge this PR pin!

**Review by rifqifarelmuhammad — APPROVED** [review]

**Pull request merged**